src/PresentationalComponents/Portfolio/portfoliocard.scss

```diff
@@ -0,0 +1,51 @@
+@import "../../App.scss";
+
+.pcard_style {
+  margin-top: 0px;
```

Comment:
What is the reason to put every option to the single line and define the values like this?
Maybe

margin: 0px;
padding: 0px;

could work, too. https://www.w3schools.com/Css/css_margin.asp